### ISSUE TITLE
status of Calamagrostis deschampsioides in Labrador

### ISSUE THREAD

**Issue by sjmeades** [Canadensys/vascan-data#2017]
[Originally posted on GoogleCode (id 2247) on 2014-08-02]

<b>(This is the template to report a data issue for Vascan. If you want to</b>
<b>report another issue, please change the template above.)</b>

<b>What is the URL of the page where the problem occurs?</b>
http://data.canadensys.net/vascan/taxon/7491

<b>What data are incorrect or missing?</b>
Calamagrostis deschampsioides listed from Labrador

<b>What data are you expecting instead?</b>
Unless you have a recent record, Calamagrostis deschampsioides is not confirmed Labrador

<b>If applicable, please provide an authoritative source.</b>
From our NL checklist, we had this listed as unconfirmed in Labrador:
[Labrador reports of Calamagrostis deschampsioides are unconfirmed. C.W. Greene (pers. comm.) reports seeing no Labrador specimens of C. dechampsioides during his revision of Calamagrostis, but Hultén (1968), Porsild &amp; Cody (1980), and Blondeau &amp; Cayouette (1987) show Labrador distributions.  Vouchers have not been located yet.]

**Comment by brouille** [comment]
[Originally posted on GoogleCode on 2014-08-04 20:20Z]

needs verif

**Comment by brouille** [comment]
I am changing the status in Labrador from native to doubtful until one can confirm the various sources. I would leave aside the report from Hultén. The dots in P&C could be in Québec as well as in Labr. (right on border) and the map is small. Blondeau & Cayouette 87: what paper, exactly?
Luc

**Comment by sjmeades** [comment]
I agree with you that /Calamagrostis deschampsioides/ should be listed 
as doubtful for Labrador, unless we get better information.Here is the 
reference to the Blondeau and Cayouette paper:

Blondeau, M., and J. Cayouette.1987.Extensions d'aire dans la flore 
vasculaire du Nouveau-Québec. /Naturalist can/. 114: 117-126.
